### PR TITLE
#BAH-973 | Vinay | Print the List View in Bed Management module

### PR DESCRIPTION
**Pull request by vinay-ThoughtWorks**

### CODE REVIEW

File: ui/app/bedmanagement/views/wardListPrint.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<style>
+    @page {
+        size: A4 landscape;
+    }
+
+    body {
+        font-family: sans-serif;
+        font-size: 10px !important;
```
Comment (bsneha90) [inline ui/app/bedmanagement/views/wardListPrint.html]: Can we avoid using important?

Comment (vinay-ThoughtWorks) [reply ui/app/bedmanagement/views/wardListPrint.html]: Done

Comment (bsneha90) [reply ui/app/bedmanagement/views/wardListPrint.html]: You have removed 'important' without any other change, does it still work?

Comment (vinay-ThoughtWorks) [reply ui/app/bedmanagement/views/wardListPrint.html]: Yes. I checked. I found no change

**Pull request merged**